traitsui/testing/tester/wx/helpers.py

```diff
@@ -133,4 +140,6 @@ def key_sequence_text_ctrl(control, interaction, delay):
     if not control.HasFocus():
         control.SetFocus()
     for char in interaction.sequence:
+        if char == '\b':
+            char = "Backspace"
```

Comment:
I did not investigate why previously `\b` did not work. Now I tried... if I changed the `SetUnicodeKey` to `SetKeyCode` in `key_click` then we don't need this special casing of `\b`. Could you confirm please?

Reply:
Ah nice, yes this works for me as well.  Do you understand why?  I was under the impression '\b' would work as a unicode character.